<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAGReaderOptionPage</class>
 <widget class="QWidget" name="MAGReaderOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input Options</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Lambda value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Micron</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="keep_names_cbx">
        <property name="text">
         <string>Don't attempt to translate into layer/datatype numbers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Micron</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dbu_le">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Keep layer names</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Database unit   </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lambda_le"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="merge_cbx">
        <property name="text">
         <string>Merge boxes into polygons</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Layout healing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Library paths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="5">
       <widget class="lay::InteractiveListWidget" name="lib_path">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="del_lib_path">
        <property name="toolTip">
         <string>Delete selected paths</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../lay/lay/layResources.qrc">
          <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="move_lib_path_down">
        <property name="toolTip">
         <string>Move selected paths down</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../lay/lay/layResources.qrc">
          <normaloff>:/down.png</normaloff>:/down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="add_lib_path">
        <property name="toolTip">
         <string>Add new a directory</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../lay/lay/layResources.qrc">
          <normaloff>:/add.png</normaloff>:/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Relative paths are resolved relative to the file read.
You can use expressions inside the path components for variable paths</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="move_lib_path_up">
        <property name="toolTip">
         <string>Move selected paths up</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../lay/lay/layResources.qrc">
          <normaloff>:/up.png</normaloff>:/up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="add_lib_path_with_choose">
        <property name="toolTip">
         <string>Add new a directory (file browser)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layer_subset_grp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Layer Subset And Layer Mapping</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="read_all_cbx">
        <property name="text">
         <string>Read all layers (additionally to the ones in the mapping table)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="10" colspan="2">
       <widget class="lay::LayerMappingWidget" name="layer_map">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lay::LayerMappingWidget</class>
   <extends>QFrame</extends>
   <header>layLayerMappingWidget.h</header>
   <container>1</container>
   <slots>
    <signal>enable_all_layers(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>lay::InteractiveListWidget</class>
   <extends>QListWidget</extends>
   <header>layWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lambda_le</tabstop>
  <tabstop>dbu_le</tabstop>
  <tabstop>keep_names_cbx</tabstop>
  <tabstop>lib_path</tabstop>
  <tabstop>add_lib_path</tabstop>
  <tabstop>add_lib_path_with_choose</tabstop>
  <tabstop>del_lib_path</tabstop>
  <tabstop>move_lib_path_up</tabstop>
  <tabstop>move_lib_path_down</tabstop>
  <tabstop>read_all_cbx</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../lay/lay/layResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>layer_map</sender>
   <signal>enable_all_layers(bool)</signal>
   <receiver>read_all_cbx</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
